app/views/notes/show.html.erb

```diff
@@ -90,14 +90,14 @@
     </table>
   <% end %>
 
-  <% if current_user && ( current_user.role == "admin" || current_user.role == "moderator") && !(@node.has_tag('locked')) %>
+  <% if current_user && current_user.can_moderate? && !(@node.has_tag('locked')) %>
     <a href="javascript:window.location.reload(true)" onClick="addTag('locked');">Lock Note</a>
   <% end %>
 
   <% if current_user && 
-    ( current_user.role == "admin" || current_user.role == "moderator" || 
-      current_user.id == @node.id || current_user.is_coauthor?(@node) ) && 
-      @node.has_tag('event') %>
++    ( current_user.can_moderate? || 
++      current_user.id == @node.id || current_user.is_coauthor?(@node) ) && 
++      @node.has_tag('event') %>
```

Comment:
Hi @unnati914. Could you please remove the `+` from these lines?

Reply:
yes sure

Reply:
done @Manasa2850